gnmi_server/server.go

```diff
@@ -388,13 +404,11 @@ func (s *Server) Set(ctx context.Context, req *gnmipb.SetRequest) (*gnmipb.SetRe
 		/* Add to Set response results. */
 		results = append(results, &res)
 	}
-	if s.config.EnableTranslibWrite {
```

Comment:
>EnableTranslibWrite [](http://example.com/codeflow?start=13&length=19)

Why removing this check?

Reply:
I move the check to line 353, when EnableTranslibWrite is false, no need to authenticate and no need to update result.